pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -1024,9 +1024,9 @@ message KustomizeReplica {
 message ListGenerator {
   repeated k8s.io.apiextensions_apiserver.pkg.apis.apiextensions.v1.JSON elements = 1;
 
-  optional string elementsYaml = 2;
+  optional ApplicationSetTemplate template = 2;
 
-  optional ApplicationSetTemplate template = 3;
+  optional string elementsYaml = 3;
 }
```

Comment:
Apparently the struct order changes the generated proto file order, which is a breaking change.